core/src/main/java/com/datastax/oss/driver/internal/core/metadata/token/DefaultReplicationStrategyFactory.java

```diff
@@ -41,6 +41,8 @@ public ReplicationStrategy newInstance(Map<String, String> replicationConfig) {
         return new SimpleReplicationStrategy(replicationConfig);
       case "org.apache.cassandra.locator.NetworkTopologyStrategy":
         return new NetworkTopologyReplicationStrategy(replicationConfig, logPrefix);
+      case "org.apache.cassandra.locator.EverywhereStrategy":
+        return new EverywhereReplicationStrategy();
```

Comment:
A drive-by fix noticed while testing this against various DSE versions.  In addition to inappropriate queries against system_virtual_schema for DSE 6.0.x (the heart of this ticket) I was also seeing exceptions for failures to find a replication strategy impl for EverywhereStrategy.  Brought in the fix from the DSE side to address this failure.